starter-code/index.js

```diff
@@ -1,10 +1,55 @@
-var SortedList = function() {};
-
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
-
-module.exports = SortedList;
+class SortedList {
+    constructor (items = []) {
+        this.items = items;
+        this.length = this.items.length;
+    }
+  
+  add(x) {
+    let i = 0;
+    
+    while (x > this.items[i] && i < this.length) {
+            i++;
+        }
+    this.length++;
+    this.items.splice(i, 0, x);
+  }
+        
+  get(x) {
+    return this.items[x - 1];
+  }
+            
+  max() {
+    return this.items[this.length - 1];
+  }
+                
+  min() {
+    return this.items[0];
+  }
+                    
+  average() {
+    if (this.length <= 0)
+        return 0;
+                        
+        return this.sum() / this.length;
+  }
+  sum() {
```

Comment:
This way is fine too but we learned .reduce(). Try doing it on that way and using fat arrow syntax.